docs/Coding-Guidelines.md

```diff
@@ -20,7 +36,7 @@ Windows-specific helpers rather than creating your own.
 
 **DO** use 4-space indentation instead of tab characters.
 
-##### Catching C++/WinRT Exceptions and HRESULT
+##### 1.1.1.0.1. Catching C++/WinRT Exceptions and HRESULT
```

Comment:
This is worse than an IP address. Perhaps have separate docs for how to write C++ code vs. C# code vs. formatting vs. how to properly configure the CRT vs. all of Reunion. I don't think merging everything into one doc is helping readability.

Reply:
Snicker. It did point out how we borked the header styles. I fixed that.

As for one doc...'twas only a matter of time. We talked about splitting out content to separate docs and this becomes more of an index/links-to-everything-else. Looks like the day has come. Splitting up...

Reply:
Ahhhh. Much better, and room to grow. Thanks Jeff!